tests/acceptance/standard/test_hooks_processor.py

```diff
@@ -0,0 +1,149 @@
+import logging
+import pytest
+import textwrap
+import time
+from typing import TYPE_CHECKING
+
+import gitlab
+from gitlab.v4.objects import ProjectHook, Project, Group
+
+from gitlabform import GitLabForm
+from gitlabform.gitlab import GitlabWrapper
```

Comment:
I think these 2 imports are not needed, right?

Reply:
yeah, I cleaned it up, sorry (very spartan ide, I keep forgetting myself)